src/lib/base/component-manager.js

```diff
@@ -64,6 +64,13 @@ const nonBubblingEventsMap = {
 // match & select "[eventType] [css selector]"
 const handlerMethodPattern = new RegExp(`^(${events.join('|')}|${Object.keys(nonBubblingEventsMap).join('|')}) (.*)`);
 
+/**
+ * Injects fallback mouse events for non-touch devices to events object.
+ */
+function injectFallbackMouseEvents(events) {
+    events['click'] = events['touchend'] || events['tap'];
```

Comment:
Will this be a problem when there are neither `touchend` nor `tap` event handlers on a component and someone clicks on it? Would it make sense to change the conditional on line 94 to also check if either `touchend` or `tap` exists?

Reply:
The initial render seems fine but I can see it _can_ cause errors after some lifecycle methods are executed. Even if it wouldn't cause any problems, we'll have a bunch of  `{ click: undefined }` lying around, which is not really neat. Thanks for pointing that out!